system/lib/compiler-rt/exception_builtins.c

```diff
@@ -0,0 +1,3 @@
+void __trap() {
+  __builtin_trap();
```

Comment:
Lets just call this file `__trap.c` for now?

Reply:
Done